tests/cypress/e2e/build-configure.cy.js

```diff
@@ -0,0 +1,53 @@
+describe('The Build Configure Page', () => {
+  it('can be reached from the index page', () => {
+    cy.visit('index.php?project=InsightExample');
+
+    cy.get('table#project_5_15').find('tbody').find('tr').first().as('build_row');
+    cy.get('@build_row').find('td').each((table_entry, col) => {
+      if (col === 2 || col === 3) {
+        // check urls in both columns under the 'Configure' header
+        cy.wrap(table_entry).find('a').as(`configure_url_${col}`).should('have.attr', 'href').and('match', /builds?\/[0-9]+\/configure/);
```

Comment:
Why do we care about verifying that the route matches a given pattern?  It does nothing to verify that the configure page actually lives at that location.  Instead, you could click on the link and expect some text to exist.  That runs into a similar issue in that the test will have to be updated if the text is changed, but at least it guarantees some amount of correctness.

Reply:
I have changed this bit to instead test the header of the page which should show `<project> - Configure`